src/components/Item.js

```diff
@@ -1,15 +1,12 @@
 import React from "react";
 
-// I removed the myStyle function and replaced it with a condition
-// changed the whole class to a function
-
 function Item(props) {
   return (
     <div style={{ textDecoration: props.list.completed && "line-through" }}>
```

Comment:
Did you understand what happens at this line ?

Reply:
I understand the fact that you're styling the <div> content with a text decoration of line-through if completed is set to true. I also understand the replacement of the function with the condition as well the change of the function.
But the syntax you've used is what is STRANGE! Kindly explain

Reply:
I understand what happens but I don't understand the syntax, kindly explain

Reply:
The syntax is a short circuit, it checks if `props.list.completed` is truthy(not just `true`) then it returns "line-through" it returns nothing otherwise. 
Another way would be to use ternary like so `props.list.completed ? "line-through" : null` this also works

Reply:
Cool. So truthy uses "&&" and true uses "?"?

Reply:
What's the different between truthy and true?